SQL/Cleanup_patches/2018-10-22-ExpireAllPasswords.sql

```diff
@@ -0,0 +1,6 @@
+-- This forces a password reset for all users the next time they login. Before
+-- this patch, all passwords changed by the front-end were stored in plaintext.
+UPDATE users SET Password_expiry='1990-01-01';
+-- Remove old plaintext passwords. Also affects old history entries when the
+-- column was named `hash` instead of `Password_hash`
+DELETE FROM history WHERE col = "Password_hash" OR col = "Password_md5" AND tbl='users';
```

Comment:
```suggestion
DELETE FROM history WHERE (col = 'Password_hash' OR col = 'Password_md5') AND tbl='users';
```

I keep forgetting this suggest features exists now

Reply:
Thanks!